site/pages/createDisruption.tsx

```diff
@@ -0,0 +1,9 @@
+const CreateDisruption = () => {
```

Comment:
For the filename, I think dashes might be nicer than camelcase for the URLs since they're a bit clearer from my perspective so maybe `create-disruption.tsx` but I'm open to discussion if everyone else prefers camelcase

Reply:
im happy either way, personally

Reply:
done